Gems/GraphCanvas/Code/Source/GraphCanvas.cpp

```diff
@@ -412,6 +412,8 @@ namespace GraphCanvas
 
     void GraphCanvasSystemComponent::PopulateTranslationDatabase()
     {
+        m_translationAssets.clear();
```

Comment:
Should this be done after the loop at the end of the function? Or possibly in OnCatalogLoaded before AZ::Data::AssetCatalogRequestBus::Events::EnumerateAssets is called?

Reply:
Good point, I clearly need more coffee this morning. It gave the impression that it worked because it basically did nothing.